src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,44 +1,82 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+
+interface Props {
+  setSelectedUser: (user: User | null) => void,
+  selectedUser: User | null;
+  selectedTodo: Todo | null;
+  closeModal: () => void;
+}
+
+export const TodoModal: React.FC<Props> = ({
+  setSelectedUser,
+  selectedUser,
+  selectedTodo,
+  closeModal,
+}) => {
+  const isTodoSelected = selectedTodo;
```

Comment:
Why not just convert selectedTodo to boolean instead where you need it? This variable gives us absolutely no info